### ISSUE TITLE
Compile error in version `0.7.1`

### ISSUE THREAD

**Issue by LouisPennachio** [Kotlin/kotlinx-kover#415]
**Describe the bug**
I'm using the Kover Gradle plugin in my `buildSrc` folder : 

```
dependencies {
    implementation("org.jetbrains.kotlinx", "kover-gradle-plugin", "0.7.1")
}
```

It works fine with version `0.7.0`, but I get a compile error with version `0.7.1`.

**Errors**
```
 Class 'kotlinx.kover.api.KoverMergedConfig' was compiled with an incompatible version of Kotlin. The binary version of its metadata is 1.8.0, expected version is 1.5.1.
```

**Expected behavior**
The Gradle task `:buildSrc:compileKotlin` should not fail.

**Reproducer**
Including the dependency to Kover Gradle plugin inside the `buildSrc` folder fails the build.

**Reports**
N/A

**Environment**
 - Kover Gradle plugin version: 0.7.1
 - Kotlin Gradle plugin: 1.8.10
 - Gradle version: 7.3.1
 - Kotlin project type: Kotlin/Android
 - Coverage Toolset (if customized in build script): Kover (no extra config)

**Comment by LouisPennachio** [comment]
The same issue occurs with version `0.7.2`

**Comment by shanshin** [comment]
Fixed in `0.7.3`

**Comment by LouisPennachio** [comment]
Thanks a lot !
